antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -119,7 +131,9 @@ def get_command(self, study_id: str, command_id: str, params: RequestParameters)
 
         try:
             index = [command.id for command in study.commands].index(command_id)  # Maybe add Try catch for this
-            return t.cast(CommandDTOAPI, study.commands[index].to_dto().to_api())
+            command = study.commands[index]
+            user_name = self._get_user_name_from_id(command.user_id)
```

Comment:
command.user_id could be None. If so don't call the DB just put None as the user_name